examples/Rules-method-maxDose-IncrementsRelativeDLTCurrent.R

```diff
@@ -0,0 +1,21 @@
+
+# Create the data
+data <- Data(x=c(0.1, 0.5, 1.5, 3, 6, 10, 10, 10),
```

Comment:
one easy way to fix these linter formatting errors often is to install the `styler` R-package and then in R studio to use the corresponding Addin to style a file. please have a try